frontend-react/src/resources/SubmissionsResource.ts

```diff
@@ -12,9 +12,9 @@ type SubmissionsResourceParams = {
 const FALLBACKDATE = "2020-01-01T00:00:00.000Z";
 
 export default class SubmissionsResource extends AuthResource {
-    readonly taskId: number = 0;
-    readonly createdAt: string = FALLBACKDATE; // format is "2022-02-01T15:11:58.200754Z"
-    readonly sendingOrg: string = "";
+    readonly submissionId: number = 0;
+    readonly timestamp: string = FALLBACKDATE; // format is "2022-02-01T15:11:58.200754Z"
+    readonly sendend: string = "";
     readonly httpStatus: number = 0;
```

Comment:
sendend is a compound expression, correct?  camelCase?

Reply:
Good catch.  This was a typo.